svcomp-benchmark/scripts/README.md

```diff
@@ -1,13 +1,14 @@
 #### Requirements
 * python-2.7
+* [RV-Match](https://runtimeverification.com/match/)
 
 #### Running the Scripts
 
 1. Install Python Dependencies
 	* `pip install subprocess32`
 
-2. Run `python svcomp-runner.py <SV-Comp benchmarks path> <Log file path>`. The results of running the RV-Match on the benchmarks are stored at the location specified by the `log file path` argument to the python process. So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like `python /home/rvmatch/sv-benchmarks/c /home/rvmatch/results`.
+2. Run `python svcomp-runner.py <SV-Comp benchmarks path> <Log file path>`. The results of running the RV-Match on the benchmarks are stored at the location specified by the `log file path` argument to the python process. So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like `python svcomp-runner.py /home/rvmatch/sv-benchmarks/c /home/rvmatch/results`.
```

Comment:
So, for instance, if the sv-benchmarks reside at "/home/rvmatch/sv-benchmarks/c", and I want the result files to be stored at "/home/rvmatch/results", the call to the python script would look like 

--> 

For instance, if the sv-benchmarks are stored in "/home/rvmatch/sv-benchmarks/c" and the result files should go in "/home/rvmatch/results", the call to the python script would look like


(removing I sounds a bit more professional)